src/workspace.ts

```diff
@@ -165,20 +165,20 @@ export class Workspace {
    * wildcard-containing patterns as needed) to return full GitHub repo
    * information for all matched repos.
    */
-  private async _determineGitHubRepos(): Promise<GitHubRepo[]> {
+  async determineGitHubRepos(): Promise<GitHubRepo[]> {
```

Comment:
I'd keep the underscore, it's still not meant to be publicized as part of the public API.